packages/cms-cli/bin/cli.js

```diff
@@ -35,15 +35,12 @@ const argv = yargs
   .middleware([setLogLevel])
   .exitProcess(false)
   .fail((msg, err, _yargs) => {
-    // Preserve stack trace.
-    if (err) throw err;
-
     if (msg) logger.error(msg);
     if (err) logErrorInstance(err);
 
     // Give command-specifc help
     console.log(_yargs.help());
```

Comment:
I do wonder if we always want to be outputting help when there is an error.

Reply:
It seemed like it would be helpful since it is context sensitive to the command that failed.  I can take it out if you think so, but I liked it

Reply:
I think outputting help here suggests that the error is an error with the syntax. I think in most cases those errors are being reported sooner and if they are not, they should be.